### PR TITLE
feat: Project status card

### LINKED ISSUE #89: Project sheet: status card in general info

### PR DESCRIPTION
**Pull request by ncomerci**
Closes #89

### CODE REVIEW

File: src/components/Projects/ProjectStatusCardWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+function prioritizedStatus(
+  project: Project,
+  update?: UpdateAttributes
+): { status: ProjectStatus | ProjectHealth; date: Date; href?: string } {
+  const { status: projectStatus, updated_at, created_at } = project
+
+  const defaultStatus = { status: projectStatus, date: updated_at || created_at }
+
+  if ([ProjectStatus.Finished, ProjectStatus.Revoked, ProjectStatus.Paused].includes(projectStatus)) {
```
Comment (andyesp) [inline src/components/Projects/ProjectStatusCardWrapper.tsx:21]: Can we give a name to these group of statuses? What do they represent?

File: src/components/Projects/ProjectStatusCardWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  project: Project
+}
+
+function prioritizedStatus(
+  project: Project,
+  update?: UpdateAttributes
+): { status: ProjectStatus | ProjectHealth; date: Date; href?: string } {
+  const { status: projectStatus, updated_at, created_at } = project
+
+  const defaultStatus = { status: projectStatus, date: updated_at || created_at }
```
Comment (andyesp) [inline src/components/Projects/ProjectStatusCardWrapper.tsx:19]: Is this variable renaming necessary? Isn't status here a project status?

File: src/components/Projects/ProjectStatusCardWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+interface Props {
+  project: Project
+}
+
+function prioritizedStatus(
+  project: Project,
+  update?: UpdateAttributes
+): { status: ProjectStatus | ProjectHealth; date: Date; href?: string } {
+  const { status: projectStatus, updated_at, created_at } = project
```
Comment (andyesp) [inline src/components/Projects/ProjectStatusCardWrapper.tsx:17]: Is this variable renaming necessary? Isn't status here a project status?

File: src/components/Projects/ProjectStatusCardWrapper.tsx (hunk 1 above)
Comment (1emu) [reply src/components/Projects/ProjectStatusCardWrapper.tsx:21]: not ongoing? vs the in progress/pending statuses

**Review by 1emu — APPROVED** [review]

File: src/components/Projects/ProjectStatusCardWrapper.tsx (hunk 1 above)
Comment (ncomerci) [reply src/components/Projects/ProjectStatusCardWrapper.tsx]: "Ongoing” is shown but has the lowest priority, that's why it is not included there

**Review by andyesp — APPROVED** [review]

**Pull request merged**